docs/book/src/topics/multitenancy.md

```diff
@@ -0,0 +1,57 @@
+# Multi-tenancy
+
+To enable single controller multi-tenancy, a different Identity can be added to the Azure Cluster that will be used as the Azure Identity when creating Azure resources related to that cluster.
+
+This is achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library. 
+
+## Service Principal Identity
+
+Once a new SP Identity is created in Azure, a new resource of AzureIdentity should be created in the managment cluster, for example
```

Comment:
Might be good to add a reference to this doc - https://azure.github.io/aad-pod-identity/docs/concepts/azureidentity/